proxy/logging/Log.cc

```diff
@@ -305,6 +306,21 @@ struct LoggingCollateContinuation : public Continuation {
   LoggingCollateContinuation() : Continuation(nullptr) { SET_HANDLER(&LoggingCollateContinuation::mainEvent); }
 };
 
+namespace
+{
+template <int (*UnsuckyUnmarshalFunc)(const char *&, char *, int)>
```

Comment:
Perhaps a more technically descriptive name here?

Reply:
Do you want UnmarshalFuncWithPtrRef ?

Reply:
I'd probably just take it out - I don't think it's worth the code overhead just to convert  a `char**` to `char*&`.

Reply:
OK but I prefer then to make the first parameter to the unmarshaling function be char **, even though it ideally should and could be const char **.  Converting function pointer types with reinterpret casts is too scary for me, I'd have to start wearing diapers to work.